source/agora/utils/Test.d

```diff
@@ -214,6 +217,12 @@ public struct WK
         {
             return wellKnownKeyByIndex(idx);
         }
+
+        /// Returns the commons budget key pairs
+        public static KeyPair CommonsBudget ()
+        {
+            return CommonsBudgetKeyPair;
+        }
```

Comment:
Why ?

Reply:
Yeah this is unnecessary. `Keys` is used for indexing, like `WK.Keys[idx]`.

Please try to understand the purpose of a struct before adding new code to it.

Reply:
I added it because there was another problem.
I'll remove it here because I've already solved it.